src/use-cases/uploadManuscript.test.ts

```diff
@@ -16,8 +16,8 @@ describe('uploadManuscript', (): void => {
     });
 
     it('adds the manuscriptFile to the in memory submission object', async (): Promise<void> => {
-        const submissions: { id: string; manuscriptFile?: {} }[] = [{ id: 'A' }];
+        const submissions: { id: string; files?: { manuscriptFile?: {} } }[] = [{ id: 'A' }];
         await uploadManuscript(submissions)(null, { id: 'A', fileSize: 40, file: {} });
-        expect(submissions[0].manuscriptFile).toBeDefined();
+        expect(submissions[0].files?.manuscriptFile).toBeDefined();
```

Comment:
TODO: check this undefined chaining notation is behaving correctly. MDN says node doesn't support and yet the code compiles and runs fine 🤷‍♂

Reply:
Tested this executing in unit tests using node v12.15.0 which is our minimum required version. All looks good.